solar_system.rb

```diff
@@ -0,0 +1,35 @@
+class SolarSystem
+
+  attr_reader :star_name, :planets
+
+  def initialize(star_name)
+    @star_name = star_name #Sol
+    @planets = []
+  end
+
+# Placing each of the planets to an array of planets (@planets) in this solar system (Sol)
+  def add_planet (planet)
+    return @planets << planet
+  end
+
+# Creating strings with each planet name and their index (+1) to create the numbered list.
+  def list_planets
+    list = []
+    @planets.each_with_index do |planet, index|
+      list << "#{index + 1}. #{planet.name.capitalize}"
+    end
+    return list
```

Comment:
This is returning an array, not a string. To make it a string, you could call `list.join("\n")`.